#! /bin/sh /usr/share/dpatch/dpatch-run
## 36-autoconf-fix-fidone.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove invalid fidone shell statements.

@DPATCH@
--- ACE_wrappers.orig/m4/ace_defines.m4
+++ ACE_wrappers/m4/ace_defines.m4
@@ -27,7 +27,7 @@
 ACE_CHECK_DEFINE($ace_def,
                  [AC_DEFINE_UNQUOTED(AS_TR_CPP([ACE_HAS_$ace_def])) $2],
                  [$3],
-                 [$4])dnl
+                 [$4])
 done
 ])
 
@@ -43,6 +43,6 @@
 ACE_CHECK_DEFINE($ace_def,
                  [$2],
                  [AC_DEFINE_UNQUOTED(AS_TR_CPP([ACE_LACKS_$ace_def])) $3],
-                 [$4])dnl
+                 [$4])
 done
 ])
